demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -83,7 +82,7 @@
 if TYPE_CHECKING:
     from demisto_sdk.commands.content_graph.objects.relationship import RelationshipData
 
-json = JSON_Handler()
+from demisto_sdk.commands.common.handlers import DEFAULT_JSON_HANDLER as json
```

Comment:
it was on line 19 before, now on 85. why not move the import up before the if statement